### PR TITLE
[DRAFT] Adds the ability to save various AI and cyborg variables between rounds

### PR DESCRIPTION
**Pull request by Daylight2**
## What Does This PR Do

Adds these options to the character creation screen:
- [x] Save default AI name
- [x] Save default cyborg name
- [x] Save default AI core display
- [x] Save default hologram
- [ ] Save default arrivals announcement
- [ ] Save default flavor text for AI and cyborgs
- [ ] Add optional pronouns for cyborgs
- [ ] Make a submenu to hold all this, since the main panel is getting quite crowded
- [ ] Make all of this global instead of per character
- [x] Fix all the checks failing

## Why It's Good For The Game
A QOL change that means AIs and borgs don't have to go trough the rigmarole of typing out the same name they always use, hunting trough the menu for their core display and hologram, and then pasting their preferred arrivals message literally every shift. It's not engaging and doesn't have any balance implications, so why not make it more convenient? Also, CRUNCH has been complaining about not having borg pronouns for a while now. 

The name change doesn't set your name to the one you choose on character creation, but autofills the namepick box, so you can still change it if you wish to have a different name this shift.

## Images of changes
- [ ] Add images of changes

## Testing
Spawned in as borg, AI. Spawned soulless borgs. Spawned ERT borgs.

## Changelog
:cl:
add: You can now change and save your default settings for AI and cyborg in the character creation menu.
add: Cyborgs can now choose pronouns for themselves.
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
Hello yes, is this the based department?

**Comment by Roachesintrenchcoat** [comment]
Yes this is the based department!

File: config/example/config.toml [hunk 1]
```diff
@@ -175,7 +175,7 @@ ipc_screens = [
 # Enable/disable the database on a whole
 sql_enabled = false
 # SQL version. If this is a mismatch, round start will be delayed
-sql_version = 56
+sql_version = 57
```
Comment (Fordoxia) [inline config/example/config.toml:178]:
```suggestion
sql_version = 58
```
We have moved on to 58 now.

**Review by Fordoxia — CHANGES_REQUESTED** [review]

**Comment by Sirryan2002** [comment]
No progress on this PR in over a month, closing as stale. You're welcome to reopen this when you wish to continue progress, good luck!

**Pull request closed without merging**